explorer/app/components/Project/common/projectTransformer.ts

```diff
@@ -41,3 +82,57 @@ export function getProjectDescription(
 ): string | undefined {
   return project?.projects[0].projectDescription;
 }
+
+/**
+ * Returns the list of contributors for the project.
+ * Will exclude any contributor with role "data curator".
+ * @param contributors - Project contributor response model return from API.
+ * @returns project contributors.
+ */
+function filterContributorsWithProjectContributors(
+  contributors: ContributorResponse[]
+): ContributorResponse[] {
+  return contributors.filter(
+    (contributor) => !isContributorDataCurator(contributor.projectRole)
+  );
+}
+
+/**
+ * Formats name from "firstName,middleName,lastName" to "firstName middleName lastName".
+ * @param commaDelimitedName
+ * @returns formatted name "firstName middleName lastName".
+ */
+function formatContributor(commaDelimitedName: string): string {
+  return commaDelimitedName.split(/[ ,]+/).join(" ");
+}
+
+/**
+ * Map key-value pair of collaborating organizations and corresponding citation.
+ * @param contributors - Project contributor response model return from API.
+ * @returns collaborating organizations keyed by their citation.
+ */
+function getCitationByCollaboratingOrganizations(
+  contributors: ContributorResponse[]
+): Map<string, number> {
+  const setOfCollaboratingOrganizations = new Set(
```

Comment:
Simplify naming here to just `collaboratingOrganizations`?

Reply:
I've simplified to `collaboratingOrganizationsSet` as per our discussion.